console/src/test/resources/templates/changeemail-email-template.txt

```diff
@@ -0,0 +1,12 @@
+Dear {name},
+
+You asked to change your e-mail address on {publicUrl}/.
+If you did not request any password update, just ignore this e-mail, you're safe (I DON'T THINK SO).
```

Comment:
e-mail update ?

Reply:
If you are reacting about the "I DON'T THINK SO", I had a doubt when I wrote it about a possible security problem, but now I think I misunderstood the workflow and there is no longer a problem if a user receive such an e-mail without having asked for it. It just means that someone else entered his e-mail in change form, but without validating it won't do anything more.